bootstrap/scripts/build.sh

```diff
@@ -136,7 +136,8 @@ ${REPOSITORY}/bootstrap/scripts/download_vm.sh
 echo "Prepare icons"
 mkdir icon-packs
 cd icon-packs
-wget http://github.com/pharo-project/pharo-icon-packs/archive/idea11.zip
+# update the commit hash as soon as you need a new version of the icons to be loaded
+wget http://github.com/pharo-project/pharo-icon-packs/archive/57fba57a02ef3b96c453fb9feba7b71c6a3e618e.zip -O idea11.zip
```

Comment:
I'm tempted of adding the icons in the Pharo repository under the /resources directory. @estebanlm ?

Reply:
Pavel: but I think we should move them directly to the repository  of Pharo
Pavel: at least it will simplify adding of the new ones
estebanlm: but not different packs
Pavel: I would say that everything will stay as now from this perspective, only the way how the default pack will be loaded will be different
estebanlm: but then you have to check for icon packs in different places, and adapt code for that
estebanlm: for now, I would just keep it like that

Reply:
Ok, in any case I'd make a tag later for this, so we do not access a raw commitish like that